adobe_vipm/flows/sync.py

```diff
@@ -304,8 +304,9 @@ def _log_agreement_lines(
 ) -> None:
     agreement_lines = []
     for line in agreement["lines"]:
-        actual_sku = models.get_adobe_sku(line["item"]["externalIds"]["vendor"])
-        agreement_lines.append((line, get_sku_with_discount_level(actual_sku, customer)))
+        if line["item"]["externalIds"]["vendor"] != "adobe-reseller-transfer":
```

Comment:
Maybe add a constant for adobe-reseller-transfer? Note that it's also used in AddResellerChangeListOrder step